addons/edi/views/edi_document_views.xml

```diff
@@ -108,6 +111,7 @@
 	  <field name="gateway_id" groups="edi.group_edi_gateway_view"/>
 	  <field name="transfer_id" groups="edi.group_edi_gateway_view"/>
 	  <field name="state"/>
+		<field name="processing_state" optional="hide"/>
```

Comment:
Nit: spacing/indenting is off, in changes below also.

Reply:
TBH the indentation is not well on original file, there are spaces and tabs mixed in original file